auth/store.go

```diff
@@ -207,17 +222,26 @@ func (as *authStore) AuthDisable() {
 	tx.Unlock()
 	b.ForceCommit()
 
-	as.enabledMu.Lock()
 	as.enabled = false
-	as.enabledMu.Unlock()
 
 	as.simpleTokensMu.Lock()
 	as.simpleTokens = make(map[string]string) // invalidate all tokens
+	as.simpleTokenKeeper.stop()
 	as.simpleTokensMu.Unlock()
 
 	plog.Noticef("Authentication disabled")
 }
 
+func (as *authStore) Close() error {
```

Comment:
I see. But I think the tokens should be invalidated in `Close()` like `AuthDisable()`. How about the lifetime of token after closing?